2022-01-18 12:15:57,009 INFO    MainThread:27394 [internal.py:wandb_internal():87] W&B internal server running at pid: 27394, started at: 2022-01-18 12:15:57.009033
2022-01-18 12:15:57,012 DEBUG   HandlerThread:27394 [handler.py:handle_request():130] handle_request: run_start
2022-01-18 12:15:57,013 INFO    WriterThread:27394 [datastore.py:open_for_write():77] open: /Users/thuydungnguyen/workspace/LAB/DQN-FL/LongND-FL-Parallel-Computing/wandb/offline-run-20220118_121552-2uv0jtzf/run-2uv0jtzf.wandb
2022-01-18 12:15:57,017 DEBUG   HandlerThread:27394 [meta.py:__init__():40] meta init
2022-01-18 12:15:57,020 DEBUG   HandlerThread:27394 [meta.py:__init__():54] meta init done
2022-01-18 12:15:57,022 DEBUG   HandlerThread:27394 [meta.py:probe():214] probe
2022-01-18 12:15:57,029 DEBUG   HandlerThread:27394 [meta.py:_setup_git():204] setup git
2022-01-18 12:15:57,073 DEBUG   HandlerThread:27394 [meta.py:_setup_git():211] setup git done
2022-01-18 12:15:57,074 DEBUG   HandlerThread:27394 [meta.py:_save_pip():58] save pip
2022-01-18 12:15:57,075 DEBUG   HandlerThread:27394 [meta.py:_save_pip():72] save pip done
2022-01-18 12:15:57,076 DEBUG   HandlerThread:27394 [meta.py:_save_conda():79] save conda
2022-01-18 12:15:57,922 WARNING MainThread:27394 [internal.py:wandb_internal():146] Internal process interrupt: 1
2022-01-18 12:15:58,061 DEBUG   HandlerThread:27394 [meta.py:_save_conda():89] save conda done
2022-01-18 12:15:58,061 DEBUG   HandlerThread:27394 [meta.py:probe():252] probe done
